kwarchart/src/main/java/com/kwarchart/android/chart/BarChart.kt

```diff
@@ -166,15 +197,20 @@ private fun DrawScope.drawGrids(count: Int, color: Color) {
 //            pathEffect = pathEffect
         )
         // Vertical lines
-        drawLine(
-                color = color,
-                start = Offset(i * mHGap, 0f),
-                end = Offset(i * mHGap, size.height),
-//            pathEffect = pathEffect
-        )
+//        drawLine(
+//                color = color,
+//                start = Offset(i * mHGap, 0f),
+//                end = Offset(i * mHGap, size.height),
+////            pathEffect = pathEffect
+//        )
```

Comment:
remove if not used

Reply:
Let me handle this one... Will fix it in HORIZONTAL BarChart PR.